actors/builtin/multisig/multisig_test.go

```diff
@@ -75,7 +76,166 @@ func TestConstruction(t *testing.T) {
 	})
 }
 
-func newAddr(t *testing.T, id uint64) addr.Address {
+func TestPropose(t *testing.T) {
+	actor := msActorHarness{multisig.MultiSigActor{}, t}
+
+	receiver := newIDAddr(t, 100)
+	anne := newIDAddr(t, 101)
+	bob := newIDAddr(t, 102)
+	chuck := newIDAddr(t, 103)
+
+	const noUnlockDuration = int64(0)
+	var sendValue = abi.NewTokenAmount(10)
+	var nilParams = abi.MethodParams{}
+	var signers = []addr.Address{anne, bob}
+
+	builder := mock.NewBuilder(context.Background(), t, receiver).WithCaller(builtin.InitActorAddr, builtin.InitActorCodeID)
+
+	t.Run("simple propose", func(t *testing.T) {
+		const numApprovals = int64(2)
+
+		rt := builder.Build()
+
+		actor.verifyConstruct(rt, numApprovals, noUnlockDuration, signers...)
+
+		rt.SetCaller(anne, builtin.AccountActorCodeID)
+		rt.ExpectValidateCallerType(builtin.AccountActorCodeID, builtin.MultisigActorCodeID)
+		actor.verifyPropose(rt, chuck, sendValue, builtin.MethodSend, nilParams)
+
+		actor.assertTransactions(rt, multisig.MultiSigTransaction{
```

Comment:
As a final touch-up, add a brief statement of intent toward the end of these tests. E.g.
- // the transaction remains awaiting second approval
- // the transaction has been sent and cleaned up
- // the final approval was ignored until the balance is sufficient
- // the transaction is not persisted